src/MatrixClientPeg.ts

```diff
@@ -49,6 +49,13 @@ export interface IMatrixClientCreds {
     freshLogin?: boolean;
 }
 
+/**
+ * Holds the global instance of the `MatrixClient` to use across the codebase
+ * (singleton). Looking for an `MatrixClient`? Just look for the
+ * `MatrixClientPeg` on the peg board. "Peg" is the literal meaning of something
+ * you hang something on. So you'll find a `MatrixClient` hanging on the
+ * `MatrixClientPeg`.
+ */
 export interface IMatrixClientPeg {
```

Comment:
While we're here, can we add docs to the `MatrixClientPeg` constant to say "you should be using a React context with access to a client rather than using this, as in a multi-account world this will not exist."